frontend/src/locales/en.json

```diff
@@ -97,6 +97,11 @@
         "inviteAgain": "Invite again"
       }
     },
+    "dashboard": {
+      "selectFilter": {
+        "all": "All"
```

Comment:
For me `clear` would be more intuitive.

I have some items selected, then I click all and the effect is, that all filters get removed. It does the right thing for sure (in a and-operation I need to remove all filters to show all), but it's visually strange.

"All" could in that sense also be interpreted as "Select all filters" instead of "Show all items".

Can also be a topic for a 2nd PR/improvements.